### PR TITLE
Adding bio change

### PR DESCRIPTION
**Pull request by smcueto**
Hey Michelle, I added a little more on your bio. Check it out!

### CODE REVIEW

**Review by mmedina3 — COMMENTED** [review]
I'm from the Westside.  😐

File: index.html [hunk 1]
```diff
@@ -27,8 +27,8 @@
 
 <body class="body">
 <div class="intro">
-   <h1>From Educator to JavaScript Developer</h1>
-   <p> </p>
+   <h1>From Educator to Software Engineer</h1>
```
Comment (mmedina3) [inline index.html:30]: 👩🏽‍💻

**Comment by smcueto** [comment]
@mmedina3 Sorry about that! Do you want me to make the changes?

**Review by mmedina3 — APPROVED** [review]

**Comment by mmedina3** [comment]
Steph changes

**Comment by smcueto** [comment]
Saw that you merged them!

**Pull request merged**